travis/cibuildwheel_before_build_linux.sh

```diff
@@ -0,0 +1,4 @@
+# Eccodes is only for CentOS 7 so the manylinux2010 image does not work
```

Comment:
For consistency add the `#/usr/bin/env bash` call

Reply:
changed